client/src/ui/Form.jsx

```diff
@@ -9,6 +9,7 @@ import './css/Form.css';
 import { Session } from '../meteor-session';
 import { Meteor } from "../meteor-shim";
```

Comment:
Just remove this line here.

Reply:
I just did and pushed the branch with the new commit!